hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -3057,8 +3062,24 @@ private void internalQueueCall(Call call, boolean blocking)
     }
   }
 
+  /**
+   * Define some states related to Handler.
+   */
+  private enum HandlerStatus {
+    /**Working for a Call.*/
+    IN_PROCESS,
+
+    /**Idle state, waiting for a new call.*/
+    IN_IDLE
+  }
```

Comment:
Do we need this enum? We are updating the status to either `IN_PROCESS` or `IN_IDLE` but not really exposing it anywhere.